CMakeLists.txt

```diff
@@ -399,6 +400,12 @@ endif()
 set_source(simdjson)
 resolve_dependency(simdjson 3.1.5)
 
+if(${VELOX_ENABLE_QAT_ZSTD_OT})
+  set_source(qatzstd)
+  resolve_dependency(qatzstd)
+  # add_definitions(-DVELOX_ENABLE_QAT_ZSTD_OT)
```

Comment:
Should the comment be removed?

Reply:
eventually yes. not now tho. want to first debug benchmarks without the plugin.

Reply:
Ah, ok. :) Then nothing else but I agree with @kgpai on the gflag solution for testing. :)